breithbarbot/cropper/2.0/config/packages/breithbarbot_cropper.yaml

```diff
@@ -0,0 +1,13 @@
+# Read the documentation: https://github.com/breithbarbot/cropper/blob/master/Resources/doc/index.md
+breithbarbot_cropper:
+    mappings:
+#        # An example of a custom mapping name
+#        user_avatar:
```

Comment:
There is not need for config in comments like this. I suggest adding some "real" config or close this PR since flex is smart enough to install a bundle without recipe.